charts/cloudnative-pg/templates/grafana-dashboard.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.monitoring.grafanaDashboard.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cnpg-dashboard
+  namespace: {{ default .Release.Namespace .Values.monitoring.grafanaDashboard.namespace }}
+  labels:
+    grafana_dashboard: "1"
```

Comment:
Label selectors can be [configured](https://github.com/grafana/helm-charts/blob/674b94682f1a2ebf8a72ca9a698f181b52c05592/charts/grafana/README.md?plain=1#L191), so please add a field to specify them defaulting to `grafana_dashboard`.

Reply:
I did not know that. Thanks a lot. I'll make the change.

Reply:
Changed to:

```yaml
  labels:
    {{ .Values.monitoring.grafanaDashboard.sidecarLabel }}: "1"
```

Reply:
Done.

Reply:
Changed to:
```yaml
  labels:
    {{ .Values.monitoring.grafanaDashboard.sidecarLabel }}: {{ .Values.monitoring.grafanaDashboard.sidecarLabelValue | quote }}
```